### ISSUE TITLE
precompilation erro

### ISSUE THREAD

**Issue by LeoK987** [JuliaData/CSV.jl#305]
I have been using CSV#master for 0.4.0, today I added the released version and I get the following error.  I tried Pkg.resolve() but it did not help.
```
julia> using CSV
[ Info: Precompiling CSV [336ed68f-0bac-5ca0-87d4-7b16caf5d00b]
ERROR: LoadError: ArgumentError: Package CSV does not have Parsers in its dependencies:
- If you have CSV checked out for development and have
  added Parsers as a dependency but haven't updated your primary
  environment's manifest file, try `Pkg.resolve()`.
- Otherwise you may need to report an issue with CSV

```

**Comment by sunbubble** [comment]
I just reported the same error. I got it working, but it's only a workaround.
Check https://github.com/JuliaData/CSV.jl/issues/304

**Comment by quinnj** [comment]
Fixed via 5009d931dce59c499ad0bf839f4df485615bfb90; just tagged a new release.
